src/components/CaptureTable.js

```diff
@@ -102,12 +112,14 @@ class CaptureTable extends Component {
     super(props);
     this.state = {
       isDetailsPaneOpen: false,
+      isFilterShown: false,
```

Comment:
Let's make this `true` by default. The user is like to use the filter every time they visit the tool, but will still have a mechanism for hiding it if they need more space for the content.